### PR TITLE
python3Packages.pycocotools: init at 2.0.2; python3Packages.seqeval: init at 1.2.2

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

Dependencies for `tf-models-official`.

cc @piegamesde 

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/seqeval/default.nix [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # tests call perl script and get stuck in there
+    "test_statistical_tests"
+    "test_by_ground_truth"
+  ];
+
+  pytestFlagsArray = [ "-vvvv" ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/seqeval/default.nix:44]: Left in from testing?

File: pkgs/development/python-modules/seqeval/default.nix [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools_scm
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/seqeval/default.nix:4]:
```suggestion
```

File: pkgs/development/python-modules/seqeval/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/development/python-modules/seqeval/default.nix:44]: Ah yeah.

File: pkgs/development/python-modules/seqeval/default.nix (hunk 2 above)
Comment (mweinelt) [reply pkgs/development/python-modules/seqeval/default.nix]: Yup, substituted the scm version stuff.

**Review by piegamesde — APPROVED** [review]

**Review by SuperSandro2000 — APPROVED** [review]

**Pull request merged**